### PR TITLE
Feat/enable nfs by default

### PR DESCRIPTION
**Pull request by nhaghighat**
## Description

This PR uses NFS by default

## Checklist

- [ ] The PR branch is set up to merge into `feat/k8s` instead of `dev`.

### CODE REVIEW

File: deploy/Makefile [hunk 1]
```diff
@@ -15,6 +15,19 @@ else
...
+	endif
+endif
+
+ifeq (${ENABLE_ROOK_NFS},)
+	ifeq (${ENABLE_ROOK_CEPH},)
+		# Storage provider is not defined, ROOK_NFS will be used by default
+		ENABLE_ROOK_NFS := TRUE
+	endif
+endif
+
```
Comment (fruttasecca) [inline deploy/Makefile:30]: Just making sure, is it still possible to opt for a hostPath storage provider>

**Comment by fruttasecca** [comment]
Assuming this is part of the `orchest-resources` target, how would users discover such options as of now? I think it currently doesn't matter that much but I would like to either avoid using env vars passing as arguments (it's something that we haven't done until now) or to take a conscious decision to do that, including either docs or help messages etc. Not against it btw.

**Comment by nhaghighat** [comment]
> Assuming this is part of the orchest-resources target, how would users discover such options as of now? I think it currently doesn't matter that much but I would like to either avoid using env vars passing as arguments (it's something that we haven't done until now) or to take a conscious decision to do that, including either docs or help messages etc. Not against it btw, and I don't think there is anything to change about this at the moment.

This is indeed part of orchest-resources target. I really did not expect to makefile grow this far, for future we should drop it and use a sophisticated tool.

**Review by fruttasecca — APPROVED** [review]
lgtm :)

**Pull request merged**